benchmark/cxx-source/CreateObjects.swift

```diff
@@ -0,0 +1,31 @@
+//===--- CreateObjects.swift ----------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// This is a simple test that creates thousands of C++ objects and does nothing
+// with them.
+
+import TestsUtils
+import CxxCreateObjects
+
+public let CreateObjects = BenchmarkInfo(
+  name: "CreateObjects",
+  runFunction: run_CreateObjects,
+  tags: [.validation, .bridging])
+
+@inline(never)
+public func run_CreateObjects(_ N: Int) {
```

Comment:
I assume this benchmark is just there to validate that C++ support is compiling fine.
I think it makes sense to disable this benchmark from the regular run, because it's not doing anything interesting.
Can you add the ".skip" benchmark tag?

Reply:
Actually, I'd like to test the performance of creating C++ objects. For example, I expect that this will be much slower after #30630 (when we actually have to call the constructor instead of just allocating an empty object).

Reply:
I see. Makes sense